lib/lint.js

```diff
@@ -1,10 +1,12 @@
 const spawn = require('cross-spawn')
-const args = process.argv.slice(3)
 
-module.exports = (path) => {
-  var result = spawn.sync(
+module.exports = ({
+  inputDir,
+  extras
+}) => {
+  const result = spawn.sync(
     './node_modules/.bin/snazzy',
-    args,
+    [`${inputDir}/**/*.js`, extras].filter(arg => !!arg),
```

Comment:
Empty string args were causing issues for us. I think specifically with `test.js` rather than this one. Thought it would be safer to just throw away empty args.